app/routes.py

```diff
@@ -1 +1,202 @@
-from flask import Blueprint
\ No newline at end of file
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, request, jsonify, make_response, abort
+from datetime import datetime
+import requests
+import os
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+#========HELPER FUNCTION=======================================
+def get_model_from_id(cls, model_id):
+    try:
+        model_id = int(model_id)
+    except ValueError:
+        return abort(make_response({"details":"Invalid data"}, 400))
+    
+    chosen_object = cls.query.get(model_id)
+
+    if chosen_object is None:
+        return abort(make_response({"msg":
+        f"Could not find {cls.__name__} with id:{model_id}"}, 404))
+    return chosen_object
+
+#===============TASKS ROUTES====================================
+@tasks_bp.route("", methods=["POST"])
+def create_task():   
+    try:
+        request_body = request.get_json()
+        new_task = Task(
+            title=request_body["title"],
+            description=request_body["description"]
+            )
+        db.session.add(new_task)
+        db.session.commit()
+
+        return jsonify({"task":new_task.to_dict()}), 201
+    
+    except KeyError:
+        return jsonify({"details":"Invalid data"}), 400
+
+@tasks_bp.route("", methods=["GET"])
+def get_all_tasks():
+    sort_query_value = request.args.get("sort")
+    
+    if sort_query_value == "asc":
+        tasks = Task.query.order_by(Task.title.asc()).all()
+    elif sort_query_value == "desc":
+        tasks = Task.query.order_by(Task.title.desc()).all()
+    else:
+        tasks = Task.query.all()
+    
+    result = []
+    for item in tasks:
+        result.append(item.to_dict())
+    return jsonify(result), 200
+
+@tasks_bp.route("/<task_id>", methods=["GET"])
+def get_one_task(task_id):
+    chosen_task = get_model_from_id(Task, task_id)
+    return jsonify({"task":chosen_task.to_dict()}), 200
+
+@tasks_bp.route("/<task_id>", methods=["PUT"])
+def update_one_task(task_id):
+    update_task = get_model_from_id(Task, task_id)
+    request_body = request.get_json()
+
+    try:
+        update_task.title = request_body["title"]
+        update_task.description = request_body["description"]
+    except KeyError:
+        return jsonify({"msg": "Missing required data"}), 400
+    
+    db.session.commit()
+
+    return jsonify({"task":update_task.to_dict()}), 200
+
+@tasks_bp.route("/<task_id>", methods=["DELETE"])
+def delete_one_task(task_id):
+    task_to_delete = get_model_from_id(Task, task_id)
+
+    db.session.delete(task_to_delete)
+    db.session.commit()
+
+    return jsonify({"details":
+    f'Task {task_to_delete.task_id} "{task_to_delete.title}" successfully deleted'})
```

Comment:
Small note, I recommend adding the code here to be consistent with rest of the routes. Even though Flask will add it for you, it helps someone reading your code understand that the same behavior is expected here.
```suggestion
    return jsonify({"details":
    f'Task {task_to_delete.task_id} "{task_to_delete.title}" successfully deleted'}), 200
```